resources/lang/da/home.php

```diff
@@ -0,0 +1,151 @@
+ *    attracting more community contributions to the core ecosystem of osu!.
+ *
+ *    osu!web is free software: you can redistribute it and/or modify
+ *    it under the terms of the Affero GNU General Public License version 3
+ *    as published by the Free Software Foundation.
+ *
+ *    osu!web is distributed WITHOUT ANY WARRANTY; without even the implied
+ *    warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *    See the GNU Affero General Public License for more details.
+ *
+ *    You should have received a copy of the GNU Affero General Public License
+ *    along with osu!web.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+return [
+    'landing' => [
+        'online' => '<strong>:players</strong> er online i øjeblikket i <strong>:games</strong> spil',
+        'peak' => 'Toppunkt: :count',
+        'players' => '<strong>:count</strong> registrerede spillere',
+
+        'download' => [
+            '_' => 'Download now',
+            'soon' => 'osu! kommer snart til flere operativsystemer',
+            'for' => 'til :os',
+            'other' => 'klik her for :os1 eller :os2',
+        ],
+
+        'slogan' => [
+            'main' => 'Gratis rytme-spil',
+            'sub' => 'rytmen er bare ét klik væk',
+        ],
+    ],
+
+    'search' => [
+        'advanced_link' => 'Avanceret søgning',
+        'button' => 'Søg',
+        'empty_result' => 'Intet fundet!',
+        'missing_query' => 'Søg nøgleord med mindst :n karakterer er nødvendigt',
+        'title' => 'Søg resultater',
+
+        'beatmapset' => [
+            'more' => ':count flere beatmap søgeresultater',
+            'more_simple' => 'Se flere beatmap søgeresultater',
+            'title' => 'Beatmaps',
+        ],
+
+        'forum_post' => [
+            'all' => 'Alle forums',
+            'link' => 'Søg på forummet',
+            'more_simple' => 'Se flere søgeresultater fra forummet',
+            'title' => 'Forum',
+
+            'label' => [
+                'forum' => 'Søg i forums',
+                'forum_children' => 'inkluder subforums',
+                'topic_id' => 'emne #',
+                'username' => 'forfatter',
+            ],
+        ],
+
+        'mode' => [
+            'all' => 'alle',
+            'beatmapset' => 'beatmap',
+            'forum_post' => 'forum',
+            'user' => 'spiller',
+            'wiki_page' => 'wiki',
+        ],
+
+        'user' => [
+            'more' => ':count flere spiller søgeresultater',
+            'more_simple' => 'Se flere spiller søgeresultater',
+            'more_hidden' => 'Spillersøgningen er begrænset til :max spillere. Prøv at lave om på din søgning.',
+            'title' => 'Spillere',
+        ],
+
+        'wiki_page' => [
+            'link' => 'Søg på wikiet',
+            'more_simple' => 'Se flere wiki resultater',
+            'title' => 'Wiki',
+        ],
+    ],
+
+    'download' => [
```

Comment:
This is the old array for the download page, check my pull request #2080 or `resources/lang/en/home.php` for the new array

Reply:
Thanks for letting me know!